src/Adapters/Auth/facebookaccountkit.js

```diff
@@ -0,0 +1,73 @@
+const crypto = require('crypto');
+const https = require('https');
+const Parse  = require('parse/node').Parse;
+
+const graphRequest = (path) => {
+  return new Promise((resolve, reject) => {
+    https.get(`https://graph.accountkit.com/v1.1/${path}`, (res) => {
+      var data = '';
+      res.on('data', (chunk) => {
+        data += chunk;
+      });
+      res.on('end', () => {
+        try {
+          data = JSON.parse(data);
+          if (data.error) {
+            // when something wrong with fb graph request (token corrupted etc.)
+            // instead of network issue
+            reject(data.error);
+          } else {
+            resolve(data);
+          }
+        } catch (e) {
+          reject(e);
+        }
+      });
+    }).on('error', function () {
+      reject('Failed to validate this access token with Facebook.');
+    });
+  });
+};
+
+function getRequestPath(authData, options) {
+  const access_token = authData.access_token, appSecret = options && options.appSecret;
+  if (appSecret) {
+    const appsecret_proof = crypto.createHmac("sha256", appSecret).update(access_token).digest('hex');
+    return `me?access_token=${access_token}&appsecret_proof=${appsecret_proof}`
+  }
+  return `me?access_token=${access_token}`;
+}
+
+function validateAppId(appIds, authData, options) {
+  if (!appIds.length) {
+    throw new Parse.Error(
+      Parse.Error.OBJECT_NOT_FOUND,
+      'Facebook app id is not configured.');
```

Comment:
Same here

Reply:
Facebook app id is the same for Account Kit login, so I'll probably go like 'Facebook app id for Account Kit'